docs/CAPS/Specifications.md

```diff
@@ -645,7 +645,7 @@ Possible values for `_map-<map>` key/value are:
 
 - For DWI: `FA` (fractional anisotropy), `MD` (mean diffusivity, also called apparent diffusion coefficient), `AD` (axial diffusivity), `RD` (radial diffusivity), `NDI` (neurite density index), `ODI` (orientation dispersion index) and `FWF` (free water fraction).
 
-- For PET: `fdg` (<sup>18</sup>F-Fluorodeoxyglucose), `av45` (<sup>18</sup>F-Florbetapir).
+- For PET: `18FFDG` (<sup>18</sup>F-Fluorodeoxyglucose), `18FAV45` (<sup>18</sup>F-Florbetapir).
```

Comment:
Not introduced by this PR but it seems peculiar that we only mention these two tracers for a function that seems generic.

Reply:
Agreed. I guess it was introduced at a time where only those tracers were available in our datasets and the documentation has not been updated to reflect the new tracers available in new studies.

Reply:
Ok, then if possible a more exhaustive list would be relevant in this case.

Reply:
I have updated the docs accordingly.